Add Flutter SDK to Operations Monitoring page (#36180)
DataDog/documentation: content/en/real_user_monitoring/operations_monitoring.md

  - [Browser (6.20.0)][1]
  - [Android (3.1.0)][2]
  - [iOS (3.1.0)][3]
  - [Flutter (3.0.0)][7]
  - [Kotlin Multiplatform (1.4.0)][4]
  - [React Native (3.0.0)][5]
  - [Roku (1.4.0)][6]

> On Flutter Web, operations route through the Browser SDK. Enable the `feature_operation_vital` experimental feature on the Browser SDK (see the Browser tab).

Reviewer: The standalone Flutter Web note here is disconnected from the Flutter list entry it describes. Flutter is the 4th item in the list, so the visual connection is easy to miss.

Two options:
- Nest it as a sub-item under the Flutter bullet:
  ```
  - [Flutter (3.0.0)][7]
      - On Flutter Web, operations route through the Browser SDK and require the `feature_operation_vital` experimental feature enabled.
  ```
- Remove it entirely — each Flutter tab already includes this note.

Author: Fixed in https://github.com/DataDog/documentation/pull/36180/commits/59f2cc7c428490cde79d2d71a7e647b5dc685aec

  - [Browser (6.20.0)][1]
  - [Android (3.1.0)][2]
  - [iOS (3.1.0)][3]
  - [Flutter (3.0.0)][7]

>       - On Flutter Web, operations route through the Browser SDK and require the `feature_operation_vital` experimental feature enabled.

Reviewer: **Grammar**: Missing "to be" — "require the feature enabled" is ungrammatical.

```suggestion
      - **Note**: On Flutter Web, operations route through the Browser SDK, which requires the `feature_operation_vital` experimental feature to be enabled.
```

```
{{% /tab %}}

{{% tab "Flutter" %}}
```dart
DatadogSdk.instance.rum?.startFeatureOperation(
    String name, {
    String? operationKey,
    Map<String, Object?> attributes = const {},
  }
)
```

> On Flutter Web, the Browser SDK must have the `feature_operation_vital` experimental feature enabled.

Reviewer: **Voice**: Passive construction. Suggested fix (applies to all three Flutter tabs):

```suggestion
To use operations on Flutter Web, enable the `feature_operation_vital` experimental feature in the Browser SDK.
```

```
{{% /tab %}}

{{% tab "Flutter" %}}
```dart
DatadogSdk.instance.rum?.startFeatureOperation(
    String name, {
    String? operationKey,
    Map<String, Object?> attributes = const {},
  }
)
```

> On Flutter Web, the underlying Browser SDK must have the `feature_operation_vital` experimental feature enabled.

Reviewer: "Underlying" adds no information here — the Browser SDK is just the Browser SDK. Same fix applies in the Stop success and Stop failure Flutter tabs.

```suggestion
On Flutter Web, the Browser SDK must have the `feature_operation_vital` experimental feature enabled.
```

{{% /tab %}}

{{% tab "Flutter" %}}

```dart
DatadogSdk.instance.rum?.succeedFeatureOperation(
    String name, {
    String? operationKey,
    Map<String, Object?> attributes = const {},
  }
)
```

> On Flutter Web, the Browser SDK must have the `feature_operation_vital` experimental feature enabled.

Reviewer: **Voice**: Passive construction.

```suggestion
To use operations on Flutter Web, enable the `feature_operation_vital` experimental feature in the Browser SDK.
```

```
{{% /tab %}}

{{% tab "Flutter" %}}

```dart
DatadogSdk.instance.rum?.failFeatureOperation(
    String name,
    RumFeatureOperationFailureReason failureReason, // .error, .abandoned, .other
    {
    String? operationKey,
    Map<String, Object?> attributes = const {},
  }
)
```

> On Flutter Web, the Browser SDK must have the `feature_operation_vital` experimental feature enabled.

Reviewer: **Voice**: Passive construction.

```suggestion
To use operations on Flutter Web, enable the `feature_operation_vital` experimental feature in the Browser SDK.
```
